requirements.txt

```diff
@@ -1,4 +1,4 @@
-mlflow>=2.0
-scikit-learn>=1.1
+scikit-learn==1.2.0
 ipykernel>=6.12
 ipython>=7.32
+flaml
```

Comment:
Do we need flaml overall? Or do you think it is a good idea to add it in the notebook or create a sub requirement for this? Thoughts?

Reply:
That's a good point. I think there are other examples that use FLAML by default (at least I saw some PRs), so this might be a good idea. This is not a big dependency. I am happy to move it inside the multi-class folder as well.

Reply:
I recall flaml is problematic to install with lightgbm installation difficult. Has this been fixed? If not, prefer to not including it. It's not used in this example anyway.